File: ZYYMRCDemo/ZYYMRCDemo/ViewController.m
//
//  ViewController.m
//  ZYYMRCDemo
//
//  Created by 郑尧元 on 2018/3/27.
//  Copyright © 2018年 郑尧元. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    NSMutableArray *array = [NSMutableArray array];
    [array addObject:@"dd"];
    NSLog(@"%@", @(array.count));
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: OSX Daily/TableView/TableView/ViewController.swift
//
//  ViewController.swift
//  TableView
//
//  Created by 郑尧元 on 2018/2/27.
//  Copyright © 2018年 郑尧元. All rights reserved.
//

import Cocoa

class ViewController: NSViewController, NSTableViewDataSource {

    let strings = ["apple", "banana", "orange"]
    
    
    
    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
    }

    override var representedObject: Any? {
        didSet {
        // Update the view, if already loaded.
        }
    }

    // MARK: DataSource
    func numberOfRows(in tableView: NSTableView) -> Int {
        return strings.count
    }
    
    func tableView(_ tableView: NSTableView, objectValueFor tableColumn: NSTableColumn?, row: Int) -> Any? {
        return strings[row]
    }
}

